root/static/scripts/common/components/EntityLink.js

```diff
@@ -196,6 +197,17 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
   let showDisambiguation = passedShowDisambiguation;
   let showIcon = passedShowIcon;
 
+  if (nameVariation === undefined &&
+    content !== undefined && typeof content !== 'string'
+  ) {
+    const errorMessage = 'Content of type ' + typeof content +
+      ' cannot be compared as a string to entity name for name variation.';
+    if (__DEV__) {
+      invariant(false, errorMessage);
```

Comment:
This is triggering where content is `null`, which is causing the failures. Maybe check `nonEmpty(content)` instead of `content !== undefined`.

Reply:
BTW, I was able to find the error like this:
 * Found which test the failing command `check target="xpath=//input[@id = 'change-matching-artists']" value=""` was in (it's in t/selenium/release-editor/The_Downward_Spiral.json5)
 * Ran that test with the browser open: `env SIR_DIR=/home/michael/code/sir ./t/selenium.js --headless=false --stay-open=true t/selenium/release-editor/The_Downward_Spiral.json5`
 * Opened the developer tools while the test was running and waited for an exception to occur